Benchmarks/BuildingSimulation/README.md

```diff
@@ -34,16 +34,25 @@ Ubuntu 20.04, but may require slight modification for other platforms.
 A Swift toolchain with support for differentiation must be installed and in your current path. We 
 recommend using one [downloaded from Swift.org](https://www.swift.org/download/) for your platform. 
 Nightly toolchain snapshots tend to have better performance, due to new optimizations and 
-architectural improvements constantly being upstreamed.
+architectural improvements constantly being upstreamed. More information on toolchain installation 
+and management can be found [here](https://passivelogic.github.io/differentiable-swift-examples/documentation/differentiableswiftexamples/setup).
 
 To build the benchmark, change into the `Swift` subdirectory and run the following:
```

Comment:
It might be cleaner to move this sentence down and in place of the `Then we can compile` one.